### PR TITLE
Update doc.go of package account

### PR DESCRIPTION
**Pull request by jimni1222**
## Proposed changes

Update doc.go of package account

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: blockchain/types/account/doc.go [hunk 1]
```diff
@@ -14,5 +14,41 @@
...
+
+There are three types of Accounts used in Klaytn.
+
+  - LegacyAccountType
+  - ExternallyOwnedAccountType
+  - SmartContractAccountType
+
+AccountCommon implements the structure and functions common to Klaytn Account, which also implements the Account interface.
+
+EOA(ExternallyOwnedAccount) and SCA(SmartContractAccount) are implemented in a structure that includes AccountCommon.
```
Comment (KimKyungup) [inline blockchain/types/account/doc.go]: `EOA(ExternallyOwnedAccount)` -> `EOA (ExternallyOwnedAccount)`
Please put a space before the parenthesis.

Comment (jimni1222) [reply blockchain/types/account/doc.go]: Space is added at there :)

File: blockchain/types/account/doc.go [hunk 2]
```diff
@@ -14,5 +14,41 @@
...
+
+LegacyAccount is implemented separately according to the account interface.
+
+
+
+Source Files
+
+Account related functions and variables are defined in the files listed below
```
Comment (KimKyungup) [inline blockchain/types/account/doc.go]: Please, mark the period at the end of line.

Comment (jimni1222) [reply blockchain/types/account/doc.go]: Period is added :) Thank you for reviewing

File: blockchain/types/account/doc.go [hunk 3]
```diff
@@ -14,5 +14,36 @@
...
+
+EOA (ExternallyOwnedAccount) and SCA (SmartContractAccount) are implemented in a structure that includes AccountCommon.
+
+LegacyAccount is implemented separately according to the account interface.
+
+Source Files
+
+Account related functions and variables are defined in the files listed below.
+  - account.go                  : Defines the types, interfaces and functions associated with the Account.
+  - account_common.go           : Data structures and functions that are common to EOA (ExternallyOwnedAccount) and SCA (SmartContractAccount) are defined as AccountCommon..
```
Comment (KimKyungup) [inline blockchain/types/account/doc.go]: `..`->`.`

File: blockchain/types/account/doc.go [hunk 4]
```diff
@@ -14,5 +14,36 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package account contains types related to Klaytn accounts.
+/*
+Package account implements Account used in Klaytn.
+
+Inside the package, the type, structure, functions, and interfaces associated with the Account are defined.
```
Comment (jeonghonah) [inline blockchain/types/account/doc.go]:
```suggestion
Inside the package, the type, structures, functions, and interfaces associated with the Account are defined.
```

File: blockchain/types/account/doc.go (hunk 3 above)
Comment (jimni1222) [reply blockchain/types/account/doc.go]: Oh, ;-; i edited mistypo!

File: blockchain/types/account/doc.go [hunk 5]
```diff
@@ -14,5 +14,36 @@
...
+AccountCommon implements the structure and functions common to Klaytn Account, which also implements the Account interface.
+
+EOA (ExternallyOwnedAccount) and SCA (SmartContractAccount) are implemented in a structure that includes AccountCommon.
+
+LegacyAccount is implemented separately according to the account interface.
+
+Source Files
+
+Account related functions and variables are defined in the files listed below.
+  - account.go                  : Defines the types, interfaces and functions associated with the Account.
```
Comment (jeonghonah) [inline blockchain/types/account/doc.go]:
```suggestion
  - account.go                  : Defines types, interfaces and functions associated with the Account.
```

**Pull request merged**